src/qt/transactiontablemodel.cpp

```diff
@@ -95,25 +95,27 @@ class TransactionTablePriv
      */
     QList<TransactionRecord> cachedWallet;
 
-    bool fQueueNotifications = false;
+    bool m_loaded = false;
+    int m_progress = 100;
     std::vector< TransactionNotification > vQueueNotifications;
 
     void NotifyTransactionChanged(const uint256 &hash, ChangeType status);
-    void ShowProgress(const std::string &title, int nProgress);
+    void DispatchNotifications();
 
     /* Query entire wallet anew from core.
      */
     void refreshWallet(interfaces::Wallet& wallet)
     {
-        qDebug() << "TransactionTablePriv::refreshWallet";
```

Comment:
Why dropped?

Reply:
What is the goal of this log? To know when `refreshWallet` is called? Seems one of those logs you add when developing, but now it's useless. I'll revert this change if you think otherwise. It's unrelated to this PR but committed since I was touching nearby code.

Reply:
> To know when `refreshWallet` is called?

Rather to know that `refreshWallet` _is_ called, for example for old wallets that were not synced for a long time, no?

Reply:
But it's called unconditionally in `TransactionTableModel` constructor.

Reply:
> But it's called unconditionally in `TransactionTableModel` constructor.

Right. But it is an implementation detail that (as we assume) is hidden from users.

Anyway, feel free to ignore this concern and mark it as resolved :)

Reply:
> But it is an implementation detail that (as we assume) is hidden from users.

Right, so why log it? Following the same idea there are many things we could also log.